.github/integration/tests/common/10_trigger_failures.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ "$STORAGETYPE" = s3notls ]; then
+if [ "$STORAGETYPE" = s3notls ] || [ "$STORAGETYPE" = posix ]; then
```

Comment:
Why this selection, there is nothing in this test that is specific to a deployment type.

Reply:
It could be that there is a different error message returned in one of the tests which causes the posix test to not pass.